subprojects/consuming-jvm-libraries/samples/code/build.gradle

```diff
@@ -1,7 +1,15 @@
+// tag::application[]
 // tag::java[]
-apply plugin : 'java' // <1>
+plugins {
+    id 'java'
+// end::java[]
+    id 'application' // <1>
+// tag::java[]
+}
```

Comment:
I didn't know you can reopen them...crazy :)

Reply:
Yes, it is an amazing feature!